doc/dev/developer_guide/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -115,48 +139,77 @@ what bunch of tests it would trigger, look at the contents of the file for
 
-    teuthology-suite -v -m smithi -c wip-rishabh-fs-test_cephfs_shell-fix -p 50 --rerun teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi
+Once the teuthology job is scheduled, the status/results for test run could 
+be checked from https://pulpito.ceph.com/
+It could be used for quickly checking out job logs, their status etc.
 
-The meaning of rest of the options is already covered in `Triggering Tests`
-section.
 
 Teuthology Archives
--------------------
+*******************
+
 Once the tests have finished running, the log for the job can be obtained by
 clicking on job ID at the Pulpito page for your tests. It's more convenient to
-download the log and then view it rather than viewing it in an internet
-browser since these logs can easily be upto size of 1 GB. What's much more
-easier is to log in to the teuthology machine again
-(``teuthology.front.sepia.ceph.com``), and access the following path::
+download the log and then view it rather than viewing it in an internet browser
+since these logs can easily be upto size of 1 GB. What's much more easier is to
+ssh into the teuthology machine again (``teuthology.front.sepia.ceph.com``), and
+access the following path::
 
     /ceph/teuthology-archive/<test-id>/<job-id>/teuthology.log
 
 For example, for above test ID path is::
 
-    /ceph/teuthology-archive/teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi/4588482/teuthology.log
+   /ceph/teuthology-archive/teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi/4588482/teuthology.log
 
 This way the log remotely can be viewed remotely without having to wait too
 much.
 
+.. note:: to access archives more conveniently ``/a/`` has been
+symbolically linked to ``/ceph/teuthology-archive/`` for example to access
+previous example, we can use something like::
+
+   /a/teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi/4588482/teuthology.log
+
+Sentry Events
+------------- 
+In 2020, Sentry, a monitoring platform was added which helps in better
+organizing and analysis of issues observed in failed teuthology jobs.  For more
+details, please see <../tests-sentry-developers-guide.rst>
+
+Killing Tests
+------------- 
+Sometimes a teuthology job might not complete running for several minutes or
+even hours after tests that were trigged have completed running and other
+er-testing-basic-smithi
+
+Let's call the argument passed to ``-r`` as test ID. It can be found
+easily in the link to the Pulpito page for the tests you triggered. For
+example, for the above test ID, the link is -
+http://pulpito.front.sepia.ceph.com/teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi/
+
+Re-running Tests
+----------------
+If a Pass ``--rerun`` option, with test ID as an argument to it, to
+``teuthology-suite`` command:: Generally, useful in cases where teuthology test
+batch has some failed/dead jobs that we might want to retrigger, we can trigger
+jobs based on their status using::
```

Comment:
_You can pass_ ``--rerun`` option, with test ID as an argument to it, to
``teuthology-suite`` command. Generally, _this is_ useful in cases where teuthology test
batch has some failed/dead jobs that we might want to retrigger. We can trigger
jobs based on their status using::